src/C1T2X_OBU.py

```diff
@@ -39,17 +40,16 @@
 c1t2x_logger_formatter = logging.Formatter("[%(asctime)s.%(msecs)03d] %(levelname)s - %(message)s", datefmt= "%d-%b-%y %H:%M:%S")
 c1t2x_logger_handler.setFormatter(c1t2x_logger_formatter)
 c1t2x_logger.addHandler(c1t2x_logger_handler)
-# start logging
+# Start logging
 c1t2x_logger.info("\n---------------------------\nStarting C1T2X OBU Logger\n---------------------------")
```

Comment:
Do we want new lines in this log statement? Seems like it should be
`"--------------------------- Starting C1T2X OBU Logger ---------------------------"`

Reply:
This splits it into a block-shape when printing. Just a formatting decision, looks fine as-is even if it uses a lot of space.